_shared/scripts/provenance.py

```
@@ -56,10 +76,19 @@
-    "> This document's phase history does not include an initial /draft — "
-    "structure was not verified against the template from origin."
-)
+def origin_untracked_note(workflow: str | None = None) -> str:
+    origin = ORIGIN_PHASE.get(workflow, "draft")
+    # ux-design has no template step; its /handoff assembles from scratch
+    if workflow == "ux-design":
+        return (
+            f"> This document's phase history does not include an initial /{origin} — "
+            "structure was not verified from origin."
+        )
+    else:
```

Ruff (0.16.2)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)